pkgs/applications/misc/zk-shell/default.nix

```diff
@@ -1,8 +1,8 @@
-{ lib, fetchFromGitHub, pythonPackages }:
+{ lib, fetchFromGitHub, python3Packages }:
 
-pythonPackages.buildPythonApplication rec {
+python3Packages.buildPythonApplication rec {
+  pname = "zk-shell";
```

Comment:
This `zk-shell` package seems to be added here by mistake.
Comes from PR #137350 from SuperSandro2000/zk-shell